src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -111,56 +158,15 @@ public void execute() {
       resetLimiter = true;
     }
 
-    if (isBackCoralStation) {
-      if (DriverStation.getAlliance().isPresent()
-          && DriverStation.getAlliance().get() == DriverStation.Alliance.Blue) {
-        // Blue Alliance
-        if (currentRobotPose.getY() <= halfWidthField) {
-          // Low Y => "Right" station for Blue
-          PID.setSetpoint(BLUE_LEFT_STATION_ANGLE);
-        } else {
-          // High Y => "Left" station for Blue
-          PID.setSetpoint(BLUE_RIGHT_STATION_ANGLE);
-        }
-      } else {
-        // Red Alliance or invalid
-        if (currentRobotPose.getY() <= halfWidthField) {
-          PID.setSetpoint(RED_LEFT_STATION_ANGLE);
-        } else {
-          PID.setSetpoint(RED_RIGHT_STATION_ANGLE);
-        }
-      }
-
-      // Feed the fixed angle into the PID
+    if (mode == TargetMode.CORAL_STATION || mode == TargetMode.REEF) {
```

Comment:
Isaac suggested we only do the coral / reef alignment if the driver is not using the controls to try turning.
So we should write `if ( Math.abs(driverController.getRightX()) > deadband && (.... || ... )`